pages/getting-started/developers.md

```diff
@@ -211,7 +211,12 @@ NOTE: If you plan on upgrading to newer versions of the Standards in the future,
 
 ## Notes on accessibility
 
-We’ve designed the Standards to support older and newer browsers through progressive enhancement, and they officially support Internet Explorer 9 and up, along with the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage, and most agency websites should be able to safely begin support at Internet Explorer 9.
+- Originally the Standards were desiged to support Internet Explorer 9 and up
+- Now we're going to support any browser above 2% usage as defined on analytics.gov
+- This currently means we support only Internet Explorer 11 and up.
+- There will be no breaking changes in older browsers until a major release, ie 2.0.
```

Comment:
"ie" should be "i.e."'... and yeah, we might want to file an issue to remind us to remove this once we actually release 2.0... Hmmm.

Reply:
Do we need to state we still support the: latest versions of Chrome, Firefox, and Safari ? 
And the last sentence, should start: However, ...